tests-lit/tests/junk_detection/05_compilation_database_escaped_path/sample.cpp

```diff
@@ -0,0 +1,21 @@
+#ifndef QT_TESTCASE_BUILDDIR
+#error "QT_TESTCASE_BUILDDIR is not defined"
+#endif
+
+int sum(int a, int b) {
+  return a + b;
+}
+
+int main() {
+  return sum(-2, 2);
+}
+
+/**
+; RUN: cd / && %CLANG_EXEC -fembed-bitcode -g -O0 -DQT_TESTCASE_BUILDDIR=/src/builds/amd64-linux-mull %s -o %s.exe
```

Comment:
Shouldn't it be `-DQT_TESTCASE_BUILDDIR="/src/builds/amd64-linux-mull"`
So that the code sees QT_TESTCASE_BUILDDIR as a C string.

You could maybe use the macro to make sure it can be used as expected, eg a simple `std::cout << QT_TESTCASE_BUILDDIR << std::endl;` would be enough.